backend/src/main/resources/db-migrations/common/v0-[common]-schema-init.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE Subscription (
+    id VARCHAR(64) NOT NULL,
+    user_id VARCHAR(9) NOT NULL,
+    email VARCHAR(50) NOT NULL,
+    registered BOOLEAN, 
+    subscribed BOOLEAN,
+    preferred_language BIGINT,
+    alert_type VARCHAR(10) NOT NULL,
```

Comment:
Presumably this will link to a code table that contains a list of all alert types.. do you think it makes sense to do this now, or in a future PR?

Reply:
in a future PR please